src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -110,9 +110,12 @@ public class TimetableSnapshot {
    * For cases where the trip pattern (sequence of stops visited) has been changed by a realtime
    * update, a Map associating the updated trip pattern with a compound key of the feed-scoped
    * trip ID and the service date.
+   * This index includes only modified trip patterns for existing trips.
+   * It does not include trip patterns for new trips created by real-time updates (extra journeys).
+   * .
    * TODO RT_AB: clarify if this is an index or the original source of truth.
    */
-  private final Map<TripIdAndServiceDate, TripPattern> realtimeAddedTripPattern;
+  private final Map<TripIdAndServiceDate, TripPattern> realTimeModifiedTripPattern;
```

Comment:
My suggestion is to call this `realTimeNewTripPatternsForModifiedTrips`. It is long and ugly but at least it is very specific and shouldn't cause any confusion. We can rename it to something simpler later after we've cleaned up the code.

Reply:
Done